lib_eio/unix/pty.ml

```diff
@@ -0,0 +1,16 @@
+type pty = {
+  masterfd : Unix.file_descr;
+  slavefd : Unix.file_descr;
```

Comment:
The public interface should use `Eio_unix.Fd.t`, and attach them to a switch (or maybe two switches?).